operator-pipeline-images/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:38
+FROM images.paas.redhat.com/exd-sp-guild-isv/base-images:f40-latest
```

Comment:
This is an internal instance of the registry that is not available outside of the Red Hat VPN and you can't use it in public GitHub projects.

Reply:
Thanks for pointing it out. I tried it without the kerberos authentication but apparently forgot to turn of my VPN before that. I will change it to the public fedora image.